docs/concepts/what-is-a-workflow-id.md

```diff
@@ -18,6 +18,6 @@ A [Workflow Id Reuse Policy](/concepts/what-is-a-workflow-id-reuse-policy) can b
 The Temporal Platform guarantees uniqueness of the Workflow Id within a [Namespace](/concepts/what-is-a-namespace) based on the Workflow Id Reuse Policy.
 
 It is not possible for a new Workflow Execution to spawn with the same Workflow Id as another Open Workflow Execution, regardless of the Workflow Id Reuse Policy.
-An attempt to spawn a Workflow Execution with a Workflow Id that is the same as the Id of a currently Open Workflow Execution results in a `Workflow execution already started` error.
+An attempt to spawn a Workflow Execution with a Workflow Id that is the same as the Id of a currently Open Workflow Execution results in a `Worklfow Execution already started` error.
```

Comment:
```suggestion
An attempt to spawn a Workflow Execution with a Workflow Id that is the same as the Id of a currently Open Workflow Execution results in a `Worklfow Execution already started` error.
```

Unfortunately, the exact error contains "Workflow execution ..."

<img width="743" alt="Screen Shot 2022-10-27 at 10 26 57 AM" src="https://user-images.githubusercontent.com/34380806/198346389-7e374436-9c37-4d42-aac6-674f9ad2ec90.png">

So, not sure if we want to update this one.